### ISSUE TITLE
hi

### ISSUE THREAD

**Issue by xuepengdong** [akireikin/BankReportAnalyzer#1]
hi

**Comment by akireikin** [comment]
hi

**Comment by xuepengdong** [comment]
Are you a PHP worker?

**Comment by akireikin** [comment]
I am.

**Comment by xuepengdong** [comment]
How old are you?

**Comment by akireikin** [comment]
I am 22 years old. You?

**Comment by xuepengdong** [comment]
26，You?

**Comment by akireikin** [comment]
You have some questions about this repo?

**Comment by xuepengdong** [comment]
No,See you have Yii, so want to talk to you

**Comment by akireikin** [comment]
Ok. What are you interested in?

**Comment by xuepengdong** [comment]
PHP, you?

**Comment by akireikin** [comment]
Me too. It is a good thing.

**Comment by xuepengdong** [comment]
I come from China, you?

**Comment by xuepengdong** [comment]
I'm going to the meeting. I'll talk back.

**Comment by akireikin** [comment]
I am from Russia. Ok. What you are working on?

**Comment by wenziyelang** [comment]
I work in Beijing, the company is the nature of the company. I just did the project: http://yuanxiao.bailitop.com/university_usa.shtml

**Comment by akireikin** [comment]
Well, nice site.

**Comment by xuepengdong** [comment]
Beijing is very good, come to invite you to dinner

**Comment by akireikin** [comment]
Thank you.

**Comment by xuepengdong** [comment]
What about the salary and treatment in Russia?

**Comment by akireikin** [comment]
I live in Moscow. The average net salary of PHP developer is 100000 roubles per month. Which is roughly equal to 20000$ per year. Treatment is relatively cheap. A lot of medical services are completely free for people.

**Comment by wenziyelang** [comment]
在北京四年工作经验的php工作者薪资是15000RMB每月，北京五环外租三居室费用是4500RMB,我租的是一个次卧1200RMB,北京医疗比较贵，感冒了如果去医院要花500RMB左右

**Comment by xuepengdong** [comment]
In Beijing four years of work experience in the PHP workers salary is 15000RMB per month, the Beijing rings outside the rent three rooms is 4500RMB, I rent is a sub 1200RMB, Beijing medical more expensive, a cold if you go to the hospital to spend 500RMB

**Comment by akireikin** [comment]
Do you enjoy your current job?

**Comment by xuepengdong** [comment]
I don't knock on the code now, only to manage PHP engineers, so not happy. But it's a little higher.

**Comment by akireikin** [comment]
You use AGILE?

**Comment by xuepengdong** [comment]
Yes, we have a version of the project a month. Each version will launch a number of features. And so on

**Comment by akireikin** [comment]
So do we. The only difference we are using 1 week long iterations.

**Comment by xuepengdong** [comment]
You are relatively fast, our company process is slow, so the iteration is relatively slow. Can you look at the items you use Yii to do on-line project?

**Comment by xuepengdong** [comment]
We use the version of the tool is Git. Project management tool is a Zen way: www.zentao.net

**Comment by xuepengdong** [comment]
Do you use the knowledge management system?

**Comment by akireikin** [comment]
We are trying to be flexible. Not everything is perfect. But we try our best. Can you clarify the question about on-line project?

We also use Git. For task management we use YouTrack https://www.jetbrains.com/youtrack/.

Nope. We just have one `doc` folder in our repository with dozen of '.md' files and some diagrams.

**Comment by xuepengdong** [comment]
I'd like to see you do the PHP project.

**Comment by akireikin** [comment]
:smile:

**Comment by xuepengdong** [comment]
I mean, I would like to see you doing the PHP project.

**Comment by xuepengdong** [comment]
You Putin boss these two days is good, very admire

**Comment by xuepengdong** [comment]
Come off work, talk back. Bye-bye。

**Comment by akireikin** [comment]
Maybe some day you will. See you around. Bye.

**Comment by xuepengdong** [comment]
Chinese military parade this year, Putin came to Beijing, but I did not see him.Bye

**Comment by akireikin** [comment]
Hi there! How are you doing?

**Comment by wenziyelang** [comment]
@akireikin, Every day in a meeting, get up at 6 o 'clock in the morning and got home at ten o 'clock at night

**Comment by wenziyelang** [comment]
My Google email before two days and mobile phone, so I didn't see you a short message, I'm very sorry @akireikin

**Comment by akireikin** [comment]
@wenziyelang It seems you are working hard. Definitely, it deserves respect. Have you got a FaceBook account or some other communication channel?

**Comment by xuepengdong** [comment]
@akireikin very sorry, very busy these days, I set the phone Google mail password expired did not receive your information, I facebook account: yii2china@gmail.com. wechat account: xiaodongxiaoyang.
